### PR TITLE
All stereo avoidance stuff I did last week

### PR DESCRIPTION
**Pull request by rmeertens**
Improved:
- pocketdrone avoidance flight
- stereo protocol only we use 
- avoidance module that takes distances in meters and angles of these measure devices and outputs a velocity or attitude ref.

### CODE REVIEW

**Comment by flixr** [comment]
Better :-) But still doesn't compile, can you please fix your conf and make sure it works?

**Comment by rmeertens** [comment]
Mmm, what doesn't compile for you???

**Comment by flixr** [comment]
The ardrone2_RM airframe in your conf...

**Comment by rmeertens** [comment]
Mmm, strange. For me it compiles.... Any idea what could be wrong?

**Comment by flixr** [comment]
well, first of all the flight plan you set doesn't exist (or isn't added in this PR)
your bebop conf is also broken...

**Comment by rmeertens** [comment]
Added the flight plan, and removed the bebop conf!

**Comment by flixr** [comment]
C`mon Roland, you guys can do better... it should be obvious that we can't merge this as it is:
it's spewing loads of warnings because you are using the old periodic_telemetry_register (which won't work), shadowing variables, have messages in there that don't exist and added stabilization_attitude defines twice in the airframe file.
Also the new files have wrong doxygen headers...

File: sw/airborne/modules/obstacle_avoidance/guidance_OA.c [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Paparazzi; see the file COPYING.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @file modules/computer_vision/opticflow/stabilization_opticflow.c
```
Comment (flixr) [inline sw/airborne/modules/obstacle_avoidance/guidance_OA.c]: please fix the doxygen headers

**Pull request closed without merging**